### ISSUE TITLE
[READY_FOR_FRESH_INDEPENDENT_AUDIT] D03|PR1116|AUD1116-001-REPAIR|V2

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1235]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
AUDIT_KIND: FRESH_INDEPENDENT_EXACT_HEAD
PRODUCT_OWNER: #1234 (stale takeover of predecessor #1111)
PRODUCT_PR: #1116
PREDECESSOR_AUDIT: #1127 CHANGES_REQUIRED (`AUD1116-001`)
TARGET_HEAD: d47cfaec3a818f2ff6d9051a7ffb5799b116fa5f
EVIDENCE_BASE_MAIN: 7c14db5f6e43d369bf859cb5c6803024c929943c
CI_RUN: 34584146176
CI_RESULT: SUCCESS
CI_JOB: 103214316385
CI_OBSERVED: workflow-budget PASS; ruff `All checks passed!`; full pytest `938 passed in 3.76s`
MERGE_TEST_REF_AT_CI: 994cf8129d2547a469855f491225fd74e1f832a3

WHY FRESH AUDIT IS REQUIRED:
- predecessor exact head `ad1f0e4c49e9f5294f2d258ef4f3eab5ac06ff1d` was independently rejected by #1127 for one live-governance defect;
- Product head moved by three narrow commits; producer #1234 cannot self-certify;
- no open exact-head/READY audit owner existed immediately before this request.

NET REPAIR DELTA FROM REJECTED HEAD:
- `configs/data/d03_ecfr_rights_provenance_source_admission_v1.json`: +2/-2;
- `tools/verify_d03_ecfr_rights_provenance_source_admission_v1.py`: +4/-4;
- `tests/test_d03_ecfr_rights_provenance_source_admission_v1.py`: +25/-0.

AUDIT TARGET / MUST VERIFY READ-ONLY:
1. Exact-head identity is still `d47cfaec...`; if head moves, stop and report stale audit.
2. `project_authority.active_execution_issue` is absent from checked-in authority and is rejected by closed-world validation if reintroduced.
3. `project_authority.terminal_execution_evidence_issue=1103` is semantically truthful: #1103 is CLOSED/YIELDED terminal HTTP-406 execution evidence, not represented as an active owner. #1230 is also CLOSED/YIELDED and must not be substituted as an active replacement.
4. `project_authority.ownership_issue=1234` truthfully names the current Product stale-takeover owner, replacing stale #1111 ownership semantics.
5. Exact #707 request/materializer paths, Git blob identities, request identity, point-in-time URL/source/family and conservative evidence tuples remain unchanged from predecessor head.
6. No rights/source-admission broadening occurred: external IBR/private/contractor/transferred-copyright/media/ambiguous provenance remain fail-closed; project makes no legal conclusion.
7. Zero-credit boundary remains exact: no payload admission, canonical/family/training/unique-loss credit, tokenizer-fit authority, optimizer updates, model training, evaluation eligibility, paid compute, foreign pretrained weights, external LLM/API data/intelligence, or final-test access.
8. Regression tests specifically reject the stale old key, bool alias for terminal evidence issue, stale predecessor ownership substitution, and #1230 substitution.
9. Exact-head shared CI `34584146176` is terminal SUCCESS and corresponds to current Product head / merge test ref.
10. Audit must not mutate Product bytes, retrieve/use final-test contents, invent payload execution, or broaden source rights.

VERDICT CONTRACT:
- `PASS_FOR_INTEGRATION` only if all checks above hold at the same exact head; include exact head + CI run + no-findings statement.
- otherwise `CHANGES_REQUIRED` with concrete finding IDs/severity and minimal repair guidance.

INDEPENDENCE:
Auditor must be different from Product producer #1234 and must not self-merge. After PASS, handoff goes to designated integrator #1172.

TRUTH_BOUNDARY:
REAL_DATA_EXECUTED_BY_AUDIT=false
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
FRESH INDEPENDENT AUDIT CONSUMED — **PASS_FOR_INTEGRATION_ECFR_RIGHTS_PROVENANCE_MECHANICS**

Auditor #1237 completed the requested exact-head audit of PR #1116 head `d47cfaec3a818f2ff6d9051a7ffb5799b116fa5f`.

`AUD1116-001` is closed. `active_execution_issue` is structurally removed and rejected by closed-world validation; #1103 is correctly bound only as CLOSED/YIELDED terminal HTTP-406 evidence, while #1230 is also CLOSED/YIELDED and is not substituted as active authority. The repaired regressions cover both stale-key and bool/int/substitution failure modes.

Exact shared CI `34584146176` is terminal SUCCESS on the repaired head. Late-bound main is `17d4eb2de03e78fefbd20a354e0e1b382dab11a8`; PR #1116 is currently open/non-draft/mergeable, and intervening main drift does not touch the eCFR rights-policy/verifier/test or pinned request surfaces.

Primary verdict: PR #1116 comment `5633473499`.
Permanent AUDIT-A: #13 comment `5633475571`.

No real eCFR payload execution, corpus admission, tokenizer-fit authority, training authority, optimizer step, learned weights, or final-test access is claimed.
